crates/tests/winrt/reference/src/lib.rs

```diff
@@ -7,7 +7,7 @@ unsafe extern "system" fn DllGetActivationFactory(
     name: Ref<HSTRING>,
     factory: OutRef<IActivationFactory>,
 ) -> HRESULT {
-    if *name == "test_reference.Reference" {
+    if name.unwrap() == "test_reference.Reference" {
```

Comment:
`Ref` still provides `Deref` so this isn't strictly necessary but illustrates using `Ref` methods directly.